__tests__/tests.test.js

```diff
@@ -91,3 +91,59 @@ describe("/api/articles/:article_id", () => {
       });
   });
 });
+describe("/api/articles/:article_id/comments", () => {
+  test("GET 200: Returns an array of all comments for the given article id, with each comment containing the properties, comment_id, votes, created_at, author, body, article_id", () => {
+    return request(app)
+      .get(`/api/articles/1/comments`)
+      .expect(200)
+      .then((response) => {
+        const comments = response.body.comments;
+        expect(comments.length).toBe(11);
+        comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            article_id: expect.any(Number),
```

Comment:
Would you expect this comment to have any article ID? You can be more specific here